content/spin/v2/manifest-reference.md

```diff
@@ -177,6 +177,7 @@ The value of each key is a table with the following fields.
 | `source`                | Required   | String or table | The Wasm module which should handle the component. This must be built to work with the application trigger. It can be in one of the following formats: | |
 |                         |            | String      | * The path to a Wasm file (relative to the manifest file) | `dist/cart.wasm` |
 |                         |            | Table       | * The URL of a Wasm file downloadable over HTTP. This must be a table containing a `url` field for the Wasm file, and a `digest` field contains a SHA256 hex digest, used to check integrity. | `{ url = "https://example.com/example.wasm", digest = "sha256:6503...2375" }` |
+|                         |            | Table (Highly Experimental)       | * The registry, package and version of a component from a registry. This experimental source configuration must be a table containing a `registry` domain field, a `package` field and a `version` field. | `{ registry = "registrytest-abcd.fermyon.app", package = "component:hello-world", version="0.0.1" }` or `ttl.sh/user/registrytest:1.0.0` |
```

Comment:
I'm not sure what you mean by this:
```
or `ttl.sh/user/registrytest:1.0.0
```

Reply:
Ah, good catch. I will bundle that up into another full example. Sorry for the confusion.

Reply:
Sorry I am still a bit confused. Is this correct?
``{ registry = "ttl.sh", package = "user:registrytest", version="1.0.0" }``

Reply:
So I think that:
```
ttl.sh/tim/wasmthing:1.0.0
```
Translates to:
```
{ registry = "ttl.sh", package = "tim:wasmthing", version="1.0.0" }
```
If that is correct, we are good to go.